docs/quick-start/node-sdk-hello-world.md

```diff
@@ -0,0 +1,266 @@
+		});
+const transactionHash = await sender.submitTransaction(transactionBuilder);
+console.log("We created the account and got the transaction id: ", transactionHash);
+```
+#### Output
+
+```
+console.log tests/src/kinAccount.intg.test.ts:65
+  We will now create a different account
+
+console.log tests/src/kinAccount.intg.test.ts:67
+  Creating account:  GDJNMB3UPXMLVMQDWCHELVDVDHK4RAINVEO2PBA55WOBFLTT6HVTCOWI
+
+console.log tests/src/kinAccount.intg.test.ts:75
+  We created the account and got the transaction id:  e2f254f9ffb1250a7ff905893d0e6d7f9662cbe39e993517ae1645a4c8bfad86
+```
+
+### Get the Details of a Transaction
+Let's print information about the last action performed.
+
+```node
+// Get info about a tx
+console.log("We can now use the client to get info about the transaction we did");
+const transaction = await client.getTransactionData(transactionHash);
+console.log("Transaction data: ", JSON.stringify(transaction));
+````
+###### Output
+
+```
+console.log tests/src/kinAccount.intg.test.ts:78
+    Transaction data:  {"fee":100,"hash":"631d022ad9b382d4e28db65ec10adb28089ed0d92c5adefc7222496837dac2d4","sequence":5926758515736577,"signatures":[{"_attributes":{"hint":{"type":"Buffer","data":[186,22,13,146]},"signature":{"type":"Buffer","data":[250,170,219,245,2,32,89,5,75,120,26,71,101,97,44,169,40,237,144,208,248,209,204,25,163,213,87,178,45,167,108,44,116,163,110,123,128,188,197,61,71,126,159,94,85,189,22,155,196,167,143,20,110,192,104,188,61,166,78,226,82,232,50,3]}}}],"source":"GAY7ZK5S5K4V3BHMJRSXTFQKRM2MQAV5YZBGMQDA2264SPF2CYGZFQCK","timestamp":"2019-07-21T13:16:45Z","type":"CreateAccountTransaction","destination":"GA375FPA747GONJQU672GNCQ4D4WUFKKW36TKIB4LXPHIHLFYFKQIFB6","startingBalance":1000,"memo":"1-anon-Test create account"}
+
+```
+
+### Send Kin
+Now that you have a destination public address, you can send Kin to the associated account. `const keypair` holds the information of the destination account. You are going to send 10 KIN. Executing this transaction will by default incur a cost of 100 FEE. (1 KIN = 10<sup>-5</sup> FEE)
+
+```node
+const transactionBuilder = await account.buildSendKin({
+			amount: 100,
+			memoText: "Hello World",
+			address: seconedKeypair.publicAddress,
+			fee: 100
+		});
+const transactionHash = account.submitTransaction(transactionBuilder);
+console.log("The transaction succeeded with the hash ", transactionHash);
+```
+
+###### Output
+
+```
+console.log tests/src/kinAccount.intg.test.ts:88
+  The transaction succeeded with the hash  b27e9a84a5ec5c2bffdd0d04bf4e1ecc3d3726c8719d3241b76e2b2521ec61ca
+```
+
+
+Not all transactions executed on the blockchain will be charged fee. To learn more about transaction fees and whitelisting, see [Whitelist](../documentation/node-sdk#transferring-kin-to-another-account-using-whitelist-service).
+TODO: VHECK LINK ADDRESS ^^
+In whitelisting transaction we will sign the transaction builder with Whitelisted account. We will show you the process:
```

Comment:
i would consider mentioning that in production the transaction will be signed and whitelisted by THEIR developer wallet/account